src/components/Dropdown/OptionsList.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import Option from './Option';
+import ClassList, { initialClass } from '../../utils/ClassList';
+
+const OptionsList = ({
+  dropdownOpen,
+  options,
+  onClick
+}) => {
+  const handleClick = value => () => onClick(value);
+
+  const getClassName = () => {
+    const classes = initialClass(['dropdown__options']);
+    const newClasses = ClassList.of([
+      dropdownOpen
+        ? 'dropdown__options--visible'
```

Comment:
hm... you're using react, no? Maybe take advantage of react's pattern for applying inline styles in js, instead of adding namespaced classes to represent state (dropdown__options--hidden)..